content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping.rst

```diff
@@ -82,17 +84,28 @@ integration. After saving, the :guilabel:`Public and Secret Keys` are generated.
 Setup in Odoo
 =============
 
+To ensure seamless Sendcloud integration with Odoo, :ref:`install
+<inventory/shipping_receiving/sendcloud-module>` and:ref:`link
+<inventory/shipping_receiving/link-sendcloud-module>` the Sendcloud shipping connector to the
+Sendcloud account. Then,:ref:`configure Odoo fields
+<inventory/shipping_receiving/sendcloud-shipping-info>` so Sendcloud can accurately pull shipping
+data to generate labels.
+
+.. _inventory/shipping_receiving/sendcloud-module:
+
 Install the Sendcloud shipping module
 -------------------------------------
 
-After the Sendcloud account is set up and configured, it's time to configure the Odoo database.
-To get started, go to Odoo's :guilabel:`Apps` module,  search for the :guilabel:`Sendcloud
-Shipping` integration, and install it.
+After the Sendcloud account is set up and configured, it's time to configure the Odoo database. To
+get started, go to Odoo's :guilabel:`Apps` module, search for the :guilabel:`Sendcloud Shipping`
```

Comment:
```suggestion
get started, go to Odoo's :guilabel:`Apps` module, search for the `Sendcloud Shipping`
```
suggesting this because we are instructing the reader to type something in, and it hasn't directly appeared on the screen/UI yet (up to this point)